src/coffee/client.coffee

```diff
@@ -220,9 +220,8 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    @_task = options.task or new RepeaterTaskQueue {}, { attempts: 50, timeout: 200, timeoutType: 'v' }
```

Comment:
> You can pass to client constructor your own Repeater to customise attempts, timeout and timeoutType.

What @Siilwyn and I are trying to say is that should be possible to customise this values.

Value should not be hard-coded.